CONTRIBUTING.md

```diff
@@ -6,9 +6,10 @@ Thank you for your interest in contributing to snarkVM! Below you can find some
 
 Please follow the instructions below when filing a pull request:
 
-- ensure that your branch is forked from the current [master](https://github.com/ProvableHQ/snarkVM/tree/master) branch
-- run `cargo fmt` before you commit; we use the `nightly` version of `rustfmt` to format the code, so you'll need to have the `nightly` toolchain installed on your machine; there's a [git hook](https://git-scm.com/docs/githooks) that ensures proper formatting before any commits can be made, and [`.rustfmt.toml`](https://github.com/ProvableHQ/snarkVM/blob/master/.rustfmt.toml) specifies some of the formatting conventions
-- run `cargo clippy --all-targets --all-features` to ensure that popular correctness and performance pitfalls are avoided
+- Ensure that your branch is forked from the current [master](https://github.com/ProvableHQ/snarkVM/tree/master) branch.
```

Comment:
We have a master branch? We should reference `staging` here

Reply:
Want to check out the PR description 🙊

Reply:
Ah, I'm ok with the change as long as it's clear that it's the branch that is in preparation for releases. "staging" is quite self explanatory there, whereas "master" may not be.

Reply:
Alright, if we're not unambiguously excited about switching to master, then I'm keeping it at staging to avoid the overhead: [5ceecd5](https://github.com/ProvableHQ/snarkVM/pull/2651/commits/5ceecd51e792a1ac208db3fffb03f5773c22dd4f)